packages/cli/yaml/yaml-schema/src/schemas/XCodeSampleSchema.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from "zod";
+
+// https://redocly.com/docs/api-reference-docs/specification-extensions/x-code-samples/
+export const XCodeSampleSchema = z.object({
+    lang: z.string(),
+    label: z.optional(z.string()),
+    source: z.string()
+});
```

Comment:
this should be moved to live in `openapi-parser` can you add a dependency to zod there and specify the schema there ?

Reply:
`yaml-schema` currently is just the fern definition schemas so adding more would muddle it up/be confusing